drivers/osdp/src/osdp_pd.c

```diff
@@ -0,0 +1,576 @@
+/*
+ * Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(osdp, CONFIG_OSDP_LOG_LEVEL);
+
+#include <stdlib.h>
+#include <string.h>
+
+#include "osdp_common.h"
+
+#define TAG "PD: "
+
+extern struct osdp osdp_ctx;
```

Comment:
I would rather see functions that return these instead of directly exposing them as global variables

Reply:
Add a getter for this.